mmv1/templates/terraform/examples/certificate_manager_public_ca_external_account_key.tf.erb

```diff
@@ -0,0 +1,4 @@
+resource "google_certificate_manager_public_ca_external_account_key" "<%= ctx[:primary_resource_id] %>" {
+  project     = "<%= ctx[:vars]['project'] %>"
+  server_type = "STAGING"
```

Comment:
Why can't we call the prod API in this test?

Reply:
I actually thought about writing two tests here, for both environments.